test/integration/image-component/default/pages/layout-raw.js

```diff
@@ -40,6 +40,17 @@ const Page = () => {
           loading="eager"
         ></Image>
       </div>
+      <div id="image-container4">
+        <Image
+          id="raw4"
+          src="/test.png"
+          width="400"
+          height="400"
```

Comment:
Should we also allow the scenario where width/height are only in the style, not the actual props?

@cramforce Thoughts?

Reply:
I'm not sure the added flexibility there would be worth the increase in complexity in the code. Plus, if you're allowed to provide height and width via the style prop, it raises the question why can't you provide it via other sources of styling like a CSS file or whatever.

Reply:
It would be difficult to detect. I think the `width`/`height` attributes should be validated to be integers, and then people can go nuts with the CSS.

Reply:
For sure. I've got another PR after this one to add extra validation on height and width, plus tests for that.